src/applications/discover-your-benefits/containers/IntroductionPage.jsx

```diff
@@ -40,9 +40,9 @@ const IntroductionPage = ({ router }) => {
           <va-link
             href="https://www.va.gov/family-and-caregiver-benefits/"
             external
-            text="Learn about family and caregiver benefits"
+            text="Learn about family and caregiver benefits (opens in a new tab)"
```

Comment:
The `external` prop that we're using for the`va-link` component should already handle adding the `(opens in a new tab)` text